fitnessllm_dataplatform/task_handler.py

```
@@ -0,0 +1,63 @@
+
+        Args:
+            options: Dictionary containing configuration options.
+                Required keys:
+                - data_source: Source of the data (e.g., STRAVA).
+
+        Raises:
+            KeyError: If required options or environment variables are missing.
+            ValueError: If data source is not supported.
+        """
+        if 'data_source' not in options:
+            raise KeyError("Missing required option: data_source")
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)